SerenityOS-Emoji.css

```
@@ -0,0 +1,7 @@
+@font-face {
+    font-display: block;
+    font-family: "SerenityOS Emoji";
+    font-style: normal;
+    src: local("SerenityOS Emoji"), url("https://linusg.github.io/serenityos-emoji-font/SerenityOS-Emoji.ttf") format("truetype") tech(color-COLRv1);
```

TIL about `local()`! I think the format and tech need to be listed for both:

```suggestion
    src: local("SerenityOS Emoji") format("truetype") tech(color-COLRv1),
         url("https://linusg.github.io/serenityos-emoji-font/SerenityOS-Emoji.ttf") format("truetype") tech(color-COLRv1);
```

but preferably I would just not hardcode the full URL here at all:

```suggestion
    src: url("SerenityOS-Emoji.ttf") format("truetype") tech(color-COLRv1);
```